lib/src/ruby/ruby.cc

```diff
@@ -1,11 +1,39 @@
 #include <facter/ruby/ruby.hpp>
+#include <facter/logging/logging.hpp>
 #include <internal/ruby/api.hpp>
 #include <internal/ruby/module.hpp>
 
 using namespace std;
 using namespace facter::facts;
 
+static const string load_puppet = "\
```

Comment:
I could try to map this to the appropriate rb API constructs, but that is super gross and probably more fragile than just using `rb_eval_string` here